### PR TITLE
Immersive card

### PR DESCRIPTION
**Pull request by zeftilldeath**
Immersive cards can look problematic a lot of the time, especially with sublinks and standfirsts contributing to the image being hidden.

This is a tweaked design that only contains a kicker and headline. By default immersive cards display as default articles. The immersive design can be turned on by switching on 'large headline' in the fronts tool.

screen shot 2018-05-11 at 09 57 05

# Before
screen shot 2018-05-11 at 09 45 52

# After
screen shot 2018-05-11 at 09 45 27

# Before
screen shot 2018-05-11 at 09 36 43

# After
screen shot 2018-05-11 at 09 36 49

### CODE REVIEW

**Comment by zeftilldeath** [comment]
@SiAdcock As you say hard to know for sure. But it definitely wouldn't show the standfirst and links. And more importantly. It could now be turned off if there was an issue.

**Comment by paperboyo** [comment]
Much needed adjustments, thanks bunch!

> And more importantly. It could now be turned off if there was an issue.
A godsend.

Some random thoughts:
– so Large headline switches immersive cards, but only for big cards or for all? only for immersive articles or all? is this logic documented somewhere? worth re-labelling Fronts Tool switch?
– if (in immersive mode) only a kicker and a headline is displayed, how does the card config in Fronts Tool look? does it allow for sublinks and stuff?
– immersive cards do not use our [HiDPI logic](https://github.com/guardian/frontend/blob/master/common/app/views/support/Profile.scala) which means that they look ropey (at times extremely so). Is there a will to fix this, too (it’s a pity that what’s supposed to be most striking visually actually looks worst quality)?

**Comment by zeftilldeath** [comment]
@paperboyo 

- Only for immersive articles, and only for the larger cards - just like the current logic.

- Unfortunately for the time being the config tool will look the same (There will be comms sent out). This is something we want to change, but this change needs to be usable by Monday.

- Completely agree would love for us to fix this. We should talk to someone who can make this happen.

**Comment by paperboyo** [comment]
Thanks, @zeftilldeath!
> if (in immersive mode) only a kicker and a headline is displayed, how does the card config in Fronts Tool look? does it allow for sublinks and stuff?

&

> Unfortunately for the time being the config tool will look the same

FYI @Reettaphant 

> We should talk to someone who can make this happen.

Not sure who is versed in this bit of code 🤔. Because the image is often resized to height, not sure [how easy](https://github.com/guardian/frontend/issues/12216#issuecomment-196869603) it would be... Hmm _...looking for a victim..._ maybe @sndrs could at least have an opinion on how feasible this is?

File: static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss [hunk 1]
```diff
@@ -1,31 +1,26 @@
...
-        background-color: $immersive-garnett-main-1;
-    }
-
-    //Todo better way to get this immersive card type to set the height when sublinks. This is fixed height enough to fit two lines of sublink
-    &.fc-item--has-sublinks-3 .fc-item__content,
-    &.fc-item--has-sublinks-2 .fc-item__content {
-        @include mq($from: tablet) {
-            padding-bottom: $gs-gutter * 3.6;
-        }
+    // TODO: remove this temporary css fix
```
Comment (SiAdcock) [inline static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss]: Is this easy to do? What's blocking doing this now?

**Review by SiAdcock — APPROVED** [review]
This is fine 🐶👍

File: static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss (hunk 1 above)
Comment (zeftilldeath) [reply static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss]: Will take a look now 🥇

Comment (zeftilldeath) [reply static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss]: Just to let you know, large headline setting hides stanfirst already and it's done with css. My assumption that scala would be a better way of hiding these elements is wrong. So this is consistent.

**Pull request merged**